compiler/src/iree/compiler/Codegen/Common/StripConfigInfoPass.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Codegen/Common/Passes.h"
+
+#define DEBUG_TYPE "iree-codegen-strip-config-info"
+
+namespace mlir::iree_compiler {
+
+#define GEN_PASS_DEF_STRIPCONFIGINFOPASS
+#include "iree/compiler/Codegen/Common/Passes.h.inc"
+
+namespace {
+class StripConfigInfoPass final
+    : public impl::StripConfigInfoPassBase<StripConfigInfoPass> {
```

Comment:
nit: `struct` will save us some typing, since we don't need to hide data members from anybody anyway -- this pass is defined in an anonymous namespace, so only this file knows its full type anyway
```suggestion
struct StripConfigInfoPass final
    : impl::StripConfigInfoPassBase<StripConfigInfoPass> {
```